ivadomed/models.py

```diff
@@ -1486,27 +1486,29 @@ def set_model_for_retrain(model_path, retrain_fraction, map_location, reset=True
 def get_model_filenames(folder_model):
     """Get trained model filenames from its folder path.
 
-    This function checks if the folder_model exists and get trained model (.pt or .onnx) and its configuration file
-    (.json) from it.
-    Note: if the model exists as .onnx, then this function returns its onnx path instead of the .pt version.
+    This function checks if the folder_model exists and get trained model path (without .pt or .onnx extension)
+    and its configuration file (.json) from it.
```

Comment:
> Tagging @joshuacwnewton because the segment_volume API is also used by SCT.

Thank you for the heads-up! 

As far as I understand, these changes will pretty much be invisible to SCT? Just because all of the models we currently list as part of `sct_deepseg` contain either an `.onnx` file or a `.pt` model but not both:

![image](https://user-images.githubusercontent.com/16181459/155384722-e9d3abca-f44f-47ab-93db-5f66fee52fbd.png)

But, I appreciate being made aware of these changes nonetheless.

Reply:
Indeed, we the current model list, those changes won't affect SCT but could we future models, depending on how they are packaged.